lua/entities/gmod_tardis_interior/modules/sh_parts.lua

```diff
@@ -14,6 +14,17 @@ ENT:AddHook("Cordon","parts",function(self,class,ent)
     if ent.TardisPart then return false end
 end)
 
+if SERVER then
+    ENT:AddHook("OnRemove","inside_entities",function(self)
+        for i,v in ipairs(ents.FindInSphere(self:GetPos(), self.metadata.Interior.ExitDistance)) do
```

Comment:
Use the same condition that the cordon system in Doors uses:

`for k,v in pairs(ents.FindInBox(self:LocalToWorld(self.mins),self:LocalToWorld(self.maxs))) do`

In-fact this really should be a change to Doors itself, maybe make the cordon system shared and do it in there

Reply:
removed from TARDIS

Reply:
added to doors, see https://github.com/MattJeanes/Doors/pull/26